schema/Readme.md

```diff
@@ -688,12 +690,22 @@ Specifies a default value if null is provided
 
 Disambiguates relationships when needed
 
-- name: _(optional)_ defines the name of the relationship
+###### Arguments
+
+- name: _(optional, except when required for disambiguation)_ defines the name of the relationship. The name needs to be disambiguated in the following cases:
```

Comment:
It's not the name that is disambiguated. The relations get disambiguated and the name is the tool to do so.

Reply:
I think general sentence about ambiguity would be great. Somethink that explains that we need to know which 2 fields belong to the same relation.